demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -134,13 +135,14 @@ def is_bump_required(self):
                 return False
             new_metadata = self.get_pack_metadata()
             new_version = new_metadata.get('currentVersion', '99.99.99')
-            master_metadata = run_command(f"git show origin/master:{self.metadata_path}")
+            master_metadata = run_command(
+                str(PurePosixPath(PureWindowsPath(f"git show origin/master:{self.metadata_path}"))))
```

Comment:
@gal-berger I think it is better to apply the path translation only on the self.metadata_path

Reply:
pushed and tested using tox in both Linux and Windows.